Dockerfile

```diff
@@ -1,59 +1,55 @@
-FROM ruby:3.1.3-bullseye AS deps
-
-ENV DEBIAN_FRONTEND=noninteractive
-ENV BUNDLE_BUILD__SASSC=--disable-march-tune-native
-
-RUN apt update -qq \
-    && apt install -qy \
-    build-essential \
-    ca-certificates \
-    curl \
-    gnupg \
-    # libmariadb-dev \ # SWITCH MYSQL: install mariadb, and remove postgresql
-    # mariadb-client \
-    postgresql-client-13 \
-    postgresql-contrib-13 \
-    && rm -rf /var/lib/apt/lists/*
-
-
-# Install Node. Requires curl, ca-certificates, gnupg from above
-ENV NODE_MAJOR=18
-RUN mkdir -p /etc/apt/keyrings && \
-    curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg && \
-    echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_${NODE_MAJOR}.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list && \
-    apt-get update && \
-    apt-get install -y nodejs && \
-    rm -rf /etc/apt/keyrings/nodesource.gpg /etc/apt/sources.list.d/nodesource.list
-
-ENV BUNDLE_JOBS=5 \
-    BUNDLE_PATH=/bundle \
-    BUNDLE_BIN=/bundle/bin \
-    GEM_HOME=/bundle
-
-
-RUN gem update --system
-RUN gem install bundler
-
-RUN bundle config set force_ruby_platform true
-
+FROM public.ecr.aws/docker/library/ruby:3.2-alpine3.19 AS runner
+
+ENV APP_HOME="/app"
+ENV BUNDLE_BIN="/bundle/bin"
+ENV DEBIAN_FRONTEND="noninteractive"
+ENV BUNDLE_BUILD__SASSC="--disable-march-tune-native"
+ENV BUNDLE_JOBS="5"
+ENV BUNDLE_PATH="/bundle"
+ENV GEM_HOME="/bundle"
+ENV BUNDLE_GEMFILE="$APP_HOME/Gemfile"
+ENV CHROME_BIN="/usr/bin/chromium"
+ENV CHROME_PATH="/usr/lib/chromium/"
 ENV PATH="${BUNDLE_BIN}:${PATH}"
 
-ENV APP_HOME /app
 WORKDIR $APP_HOME
 
+# Update Gems first...only re-do this when bundler version changes.
+# If we need to force a gem update we'll have to `--no-cache` or add a file we can copy in
+RUN gem update --system && gem install bundler:2.3.25
 
-FROM deps as dev
-
-CMD ["tail", "-f", "/dev/null"]
-
-FROM deps as runner
-
-COPY Gemfile Gemfile.lock $APP_HOME/
-ENV BUNDLE_GEMFILE=$APP_HOME/Gemfile
-
-RUN bundle check || bundle install
-
-COPY . $APP_HOME/
+COPY ./server/apk-updated-date.txt .
+RUN apk add --update --no-cache \
+    alpine-sdk \
+    ca-certificates \
+    curl \
+    bash \
+    chromium \
+    fontconfig \
+    graphviz \
+    gcompat \
+    imagemagick \
+    imagemagick-jpeg \
+    imagemagick-pdf \
+    msttcorefonts-installer \
+    nodejs \
+    npm \
+    pandoc \
+    libpq-dev \
+    postgresql13-dev \
+    vips-dev \
+    && rm ./apk-updated-date.txt \
+    && npm install -g pm2@4.5.6
```

Comment:
Drop font libraries, `pandoc`, `chrome`, 

Do we need to upgrade `postgresql13` or still latest library?